Assets/Scripts/TCPToolkit/TCPClient.cs

```diff
@@ -216,7 +214,7 @@ private void ReceivingThread(object streamObj)
                 }
             }
 #if DEBUG_LOG
-            Debug.Log("State at client receiving thread exit : Active endpoint ? " + m_activeEndpoint.ToString() + ", Exit signal ?" + m_exitSignal + ", Buffer offset :" + bufferOffset);
+            Debug.Log("State at client receiving thread exit : Active endpoint ? " + m_activeEndpoint.ToString() + ", Exit signal ?" + m_exitSignal);
```

Comment:
Didn't say anything about this yesterday but concatenation of string with "+" normally is heavier to process than a good old String.Format("State at client receiving thread exit : Active endpoint ? {0}, Exit signal ? {1}", m_activeEndpoint.ToString(), m_exitSignal)

Reply:
Apparently, I'm very wrong in c# as seen [here](https://stackoverflow.com/questions/925423/is-it-better-practice-to-use-string-format-over-string-concatenation-in-java). Therefore, I'll shut up and approve this PR.